src/test/java/gridTest/BookDepositoryGridTest.java

```diff
@@ -0,0 +1,60 @@
+package gridTest;
+
+import static constants.Constants.BOOK_PRODUCT_URL;
+import static constants.Constants.BOOK_TITLE;
+import static constants.Constants.HOME_PAGE_TITLE;
+import static constants.Constants.SEARCH_PAGE_TITLE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import desktop.pageFactory.PageFactoryManager;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+@Execution(ExecutionMode.CONCURRENT)
+public class BookDepositoryGridTest {
+
+    private WebDriver webDriver;
+    private PageFactoryManager pageFactoryManager;
+    String hubURL =  "http://192.168.1.215:4444/wd/hub";
```

Comment:
The same comment about formatting here - you can use the `Alt+Ctrl+L` shortcut in Windows or `Code -> Reformat Code` dropdown menu in IntelliJ idea to remove extra spaces or to add missing ones (e.g. on line 34). This command should be used for all other classes as well